src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((result, style) => {
+    const [prop, value] = style.split(':').map((x) => x.trim());
```

Comment:
The destructuring assignment is used correctly to split the property and value. However, consider handling cases where a style declaration might have multiple colons, as this would currently take only the first part as the property and the second part as the value, which might not be correct for all CSS properties.